stdlib/builtins.pyi

```diff
@@ -535,7 +538,7 @@ class str(Sequence[str]):
     def title(self: LiteralString) -> LiteralString: ...
     @overload
     def title(self) -> str: ...  # type: ignore[misc]
-    def translate(self, __table: Mapping[int, int | str | None] | Sequence[int | str | None]) -> str: ...
+    def translate(self, __table: _TranslateTable) -> str: ...
```

Comment:
Or maybe `_typeshed.SupportsGetItem`? https://github.com/python/typeshed/blob/0259068ad6bc814a510009cade00fd961b5d0355/stdlib/_typeshed/__init__.pyi#L121

Note, that variance there is different.

Reply:
I had this example in mind:

```python
>>> class Some:
...    def __getitem__(self, key):
...        return MyInt(1)
... 
>>> 'a'.translate(Some())
'\x01'
```

Reply:
Oh, `SupportsGetItem` is also a `Container`, never mind.